Source/GR32.pas

```diff
@@ -6361,7 +6361,11 @@ procedure TCustomBitmap32.LoadFromFile(const FileName: string);
 {$endif LOADFROMSTREAM}
 begin
 {$ifndef FPC}
+  {$if CompilerVersion >= 31.0}

```

Comment:
For consistence we probably should be using `COMPILERRX1_UP` from GR32_Compiler.inc but since all these `XXX_UP` defines have been unnecessary since around Delphi 6 I see no reason to use them going forward.